etna/core/blocks/image.py

```diff
@@ -40,47 +41,8 @@ def __init__(
         super().__init__(required=required, help_text=help_text, **kwargs)
 
     def get_api_representation(self, value, context=None):
-        if value:
-            jpeg_image = value.get_rendition(
-                f"{self.rendition_size}|format-jpeg|jpegquality-{self.jpeg_quality}"
-            )
-            webp_image = value.get_rendition(
-                f"{self.rendition_size}|format-webp|webpquality-{self.webp_quality}"
-            )
-
-            return {
-                "id": value.id,
-                "title": value.title,
-                "image_jpeg": {
-                    "url": jpeg_image.url,
-                    "full_url": jpeg_image.full_url,
-                    "width": jpeg_image.width,
-                    "height": jpeg_image.height,
-                },
-                "image_webp": {
-                    "url": webp_image.url,
-                    "full_url": webp_image.full_url,
-                    "width": webp_image.width,
-                    "height": webp_image.height,
-                },
-                "transcript": (
-                    {
-                        "heading": value.get_transcription_heading_display(),
-                        "text": value.transcription,
-                    }
-                    if value.transcription
-                    else None
-                ),
-                "translation": (
-                    {
-                        "heading": value.get_translation_heading_display(),
-                        "text": value.translation,
-                    }
-                    if value.translation
-                    else None
-                ),
-            }
-        return None
+        serializer = DetailedImageSerializer()
```

Comment:
```suggestion
        serializer = DetailedImageSerializer(rendition_size="fill-900x900")
```

Is this what we need?